badaas.yml.example

```diff
@@ -1,45 +1,61 @@
-server:
```

Comment:
Rename this file to `badaas.example.yml` to keep syntax highlighting and improve readability.